Altis_Life.Altis/config/Config_SpyGlass.hpp

```diff
@@ -10,7 +10,7 @@ class SpyGlass {
-    LIFE_Functions[] = {"life_fnc_aan","life_fnc_aan_meta","life_fnc_acctype","life_fnc_acctype_meta","life_fnc_actionkeyhandler","life_fnc_actionkeyhandler_meta","life_fnc_addvehicle2chain","life_fnc_addvehicle2chain_meta","life_fnc_admincompensate","life_fnc_admincompensate_meta","life_fnc_admindebugcon","life_fnc_admindebugcon_meta","life_fnc_adminfreeze","life_fnc_adminfreeze_meta","life_fnc_admingetid","life_fnc_admingetid_meta","life_fnc_admingodmode","life_fnc_admingodmode_meta","life_fnc_adminid","life_fnc_adminid_meta","life_fnc_admininfo","life_fnc_admininfo_meta","life_fnc_adminmarkers","life_fnc_adminmarkers_meta","life_fnc_adminmenu","life_fnc_adminmenu_meta","life_fnc_adminquery","life_fnc_adminquery_meta","life_fnc_adminspectate","life_fnc_adminspectate_meta","life_fnc_adminteleport","life_fnc_adminteleport_meta","life_fnc_admintphere","life_fnc_admintphere_meta","life_fnc_animsync","life_fnc_animsync_meta","life_fnc_arrestaction","life_fnc_arrestaction_meta","life_fnc_atmmenu","life_fnc_atmmenu_meta","life_fnc_bankdeposit","life_fnc_bankdeposit_meta","life_fnc_banktransfer","life_fnc_banktransfer_meta","life_fnc_bankwithdraw","life_fnc_bankwithdraw_meta","life_fnc_blastingcharge","life_fnc_blastingcharge_meta","life_fnc_boltcutter","life_fnc_boltcutter_meta","life_fnc_bountyreceive","life_fnc_bountyreceive_meta","life_fnc_broadcast","life_fnc_broadcast_meta","life_fnc_buyclothes","life_fnc_buyclothes_meta","life_fnc_buyhouse","life_fnc_buyhouse_meta","life_fnc_buyhousegarage","life_fnc_buyhousegarage_meta","life_fnc_buylicense","life_fnc_buylicense_meta","life_fnc_calweightdiff","life_fnc_calweightdiff_meta","life_fnc_capturehideout","life_fnc_capturehideout_meta","life_fnc_catchfish","life_fnc_catchfish_meta","life_fnc_catchturtle","life_fnc_catchturtle_meta","life_fnc_cellphone","life_fnc_cellphone_meta","life_fnc_changeclothes","life_fnc_changeclothes_meta","life_fnc_chopshopmenu","life_fnc_chopshopmenu_meta","life_fnc_chopshopselection","life_fnc_chopshopselection_meta","life_fnc_chopshopsell","life_fnc_chopshopsell_meta","life_fnc_civloadout","life_fnc_civloadout_meta","life_fnc_civmarkers","life_fnc_civmarkers_meta","life_fnc_clearvehicleammo","life_fnc_clearvehicleammo_meta","life_fnc_clothingfilter","life_fnc_clothingfilter_meta","life_fnc_clothingmenu","life_fnc_clothingmenu_meta","life_fnc_colorvehicle","life_fnc_colorvehicle_meta","life_fnc_containerinvsearch","life_fnc_containerinvsearch_meta","life_fnc_containermenu","life_fnc_containermenu_meta","life_fnc_copbreakdoor","life_fnc_copbreakdoor_meta","life_fnc_cophouseowner","life_fnc_cophouseowner_meta","life_fnc_copinteractionmenu","life_fnc_copinteractionmenu_meta","life_fnc_coplights","life_fnc_co
```

Comment:
`life_fnc_vehicleKind` is **not** alphabetically sorted and it is **not** in lower-case. This is why I uploaded that [Config_SpyGlass.zip](https://github.com/AsYetUntitled/Framework/files/502483/Config_SpyGlass.zip) file because I did it for you.